src/main/java/frc/robot/notification/ILedIO.java

```diff
@@ -0,0 +1,9 @@
+package frc.robot.notification;
+
+public interface ILedIO {
+  
+  public void setSpark(double sparkValue);
```

Comment:
We should comminucate to the hardware abstraction to change the color, not set its motor controller (the hardware anstraction can worry about how it sets the color and how to switch to a given color)